src/commands/cicdPipeline.ts

```diff
@@ -0,0 +1,68 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+import { CloudExplorerV1 } from 'vscode-kubernetes-tools-api';
+
+const deployToAzureExtensionId = 'ms-vscode-deploy-azure.azure-deploy';
```

Comment:
💡 I believe this can also reside inside : `isDeployToAzureExtensionInstalled` Done need to be class level `const`.